js/models/conversations.js

```diff
@@ -653,6 +663,11 @@
                   return sessionCipher.closeOpenSessionForDevice();
               }
             });
+        }).catch(function(error) {
```

Comment:
For `getProfile()` and all of the `safeXX` methods, we never reject, instead defaulting to a reasonable result. In the case of `getProfile()` we just don't update the user's profile. In the case of the trust checks, we return the default value.